### DISCUSSION TITLE
Sqlite db file Example to use on loading pre existing database.

### DISCUSSION THREAD

**Discussion by jcdevz-dev** [Nozbe/WatermelonDB#1550] [Q&A] [answered]
What would a SQLite dbfile looks like from the examples of making schemas on the documents? I am confused on what should I use for the data types. Can I use **TEXT** or **VARCHAR(??)** for '**strings** ' and  **INTEGER** or **NUMERIC** for '**number**' ?

**Comment by radex** [comment, marked as answer]
It doesn't matter what you use, SQLite doesn't have real types, only type hints.

**Comment by jcdevz-dev** [comment]
Thanks for clarification
